cli/functionary/environment.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+import click
+import requests
+
+from .config import get_config_value, save_config_value
+
+
+def get_environment_list():
```

Comment:
For these sorts of helper functions that are only intended for use internally in the same file, you can name them with a prepended underscore to designate them as such and then the docstring can just be the the brief description.  So like:

```
def _get_environments():
    """Helper function to get the list of available environment"""
```

There's nothing wrong with having the more verbose docstring, just that it's not required in this situation.  If you decide to keep it, your `Returns` is incorrect, as this appears to return a list of environments.